### PR TITLE
DE23223 Improved contrast of focus state on overflow and change image buttons

### PR DESCRIPTION
**Pull request by trevorjonesd2l**

### CODE REVIEW

File: d2l-course-tile-styles.html [hunk 1]
```diff
@@ -172,8 +172,8 @@
 			display: none;
 		}
 		.menu-item:hover, .menu-item:focus {
-			color: rgba(255,255,255,0.8);
-			background: rgba(0,0,0,0.7);
+			color: rgba(255,255,255,1);
```
Comment (jstefaniuk-d2l) [inline d2l-course-tile-styles.html]: is this now just "color: white"?

Comment (trevorjonesd2l) [inline d2l-course-tile-styles.html]: tis

**Comment by jstefaniuk-d2l** [comment]
code LGTM

**Comment by cgalvind2l** [comment]
I'll do the test thing.

**Comment by cgalvind2l** [comment]
LGTM

**Pull request merged**